### Eclipse Workspace Patch 1.0
#P SIIF
Index: src/ar/com/siif/fachada/FiscalizacionFachada.java
===================================================================
--- src/ar/com/siif/fachada/FiscalizacionFachada.java	(revision 325)
+++ src/ar/com/siif/fachada/FiscalizacionFachada.java	(working copy)
@@ -10,7 +10,6 @@
 import ar.com.siif.dto.SubImporteDTO;
 import ar.com.siif.negocio.Entidad;
 import ar.com.siif.negocio.Fiscalizacion;
-import ar.com.siif.negocio.Rodal;
 import ar.com.siif.negocio.TipoProducto;
 import ar.com.siif.negocio.Usuario;
 import ar.com.siif.negocio.exception.NegocioException;
@@ -122,7 +121,9 @@
 
 	public void altaFiscalizacion(FiscalizacionDTO fiscalizacionDTO, List<MuestraDTO> muestrasDTO)
 	{
-		Rodal rodal = ubicacionFachada.getRodal(fiscalizacionDTO.getRodal().getId());
+		Ubicacion ubicacion = new Ubicacion(ubicacionFachada);
+		ubicacion.setUbicacion(fiscalizacionDTO);
+		
 		Entidad productorForestal = entidadFachada.getEntidad(fiscalizacionDTO
 				.getProductorForestal().getId());
 		Entidad oficinaForestal = entidadFachada.getEntidad(fiscalizacionDTO.getOficinaAlta()
@@ -132,7 +133,7 @@
 		Usuario usuario = usuarioFachada.getUsuario(fiscalizacionDTO.getUsuario().getId());
 
 		Fiscalizacion fiscalizacion = ProviderDominio.getFiscalizacion(fiscalizacionDTO,
-				muestrasDTO, rodal, productorForestal, oficinaForestal, tipoProducto, usuario);
+				muestrasDTO, ubicacion, productorForestal, oficinaForestal, tipoProducto, usuario);
 		fiscalizacionDAO.altaFiscalizacion(fiscalizacion);
 	}
 
Index: src/ar/com/siif/struts/actions/forms/FiscalizacionForm.java
===================================================================
--- src/ar/com/siif/struts/actions/forms/FiscalizacionForm.java	(revision 285)
+++ src/ar/com/siif/struts/actions/forms/FiscalizacionForm.java	(working copy)
@@ -110,7 +110,7 @@
 				Long.toString(fiscalizacionDTO.getOficinaAlta().getId()), "Oficina", error);
 
 		ok15 = Validator.validarComboRequerido("-1",
-				Long.toString(fiscalizacionDTO.getRodal().getId()), "Rodal", error);
+				Long.toString(fiscalizacionDTO.getPmf().getId()), "Plan de Manejo Forestal", error);
 
 		//VALIDACIONES FISCALIZACION
 		return ok2 && ok3 && ok4 && ok5 && ok6 && ok7 && ok8 && ok9 && ok10 && ok11 && ok12 && ok13
Index: src/ar/com/siif/negocio/Fiscalizacion.java
===================================================================
--- src/ar/com/siif/negocio/Fiscalizacion.java	(revision 285)
+++ src/ar/com/siif/negocio/Fiscalizacion.java	(working copy)
@@ -45,10 +45,25 @@
 	@ManyToOne()
 	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
 	@JoinColumn(name = "rodal_fk")
-	private Rodal rodal;	
+	private Rodal rodal;
+
+	@ManyToOne()
+	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
+	@JoinColumn(name = "marcacion_fk")
+	private Marcacion marcacion;
 	
 	@ManyToOne()
 	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
+	@JoinColumn(name = "tranzon_fk")
+	private Tranzon tranzon;
+	
+	@ManyToOne()
+	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
+	@JoinColumn(name = "pmf_fk")
+	private PMF pmf;
+	
+	@ManyToOne()
+	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
 	@JoinColumn(name = "tipoProducto_fk")
 	private TipoProducto tipoProducto;
 
@@ -170,4 +185,27 @@
 		this.oficinaAlta = oficinaAlta;
 	}
 
+	public Marcacion getMarcacion() {
+		return marcacion;
+	}
+
+	public void setMarcacion(Marcacion marcacion) {
+		this.marcacion = marcacion;
+	}
+
+	public Tranzon getTranzon() {
+		return tranzon;
+	}
+
+	public void setTranzon(Tranzon tranzon) {
+		this.tranzon = tranzon;
+	}
+
+	public PMF getPmf() {
+		return pmf;
+	}
+
+	public void setPmf(PMF pmf) {
+		this.pmf = pmf;
+	}
 }
Index: src/ar/com/siif/struts/actions/FiscalizacionAction.java
===================================================================
--- src/ar/com/siif/struts/actions/FiscalizacionAction.java	(revision 299)
+++ src/ar/com/siif/struts/actions/FiscalizacionAction.java	(working copy)
@@ -126,13 +126,8 @@
 		String strForward = "exitoRecuperarTiposDeEntidadParaFiscalizacionesAModificar";
 
 		try {
-			UsuarioDTO usuario = (UsuarioDTO) request.getSession().getAttribute(
-					Constantes.USER_LABEL_SESSION);
 			WebApplicationContext ctx = getWebApplicationContext();
 
-			IRolFachada rolFachada = (IRolFachada) ctx.getBean("rolFachada");
-			//rolFachada.verificarMenu(Constantes.MODIFICACION_FISCALIZACION_MENU, usuario.getRol());
-
 			IEntidadFachada entidadFachada = (IEntidadFachada) ctx.getBean("entidadFachada");
 
 			String idTipoDeEntidad = request.getParameter("idTipoDeEntidad");
@@ -196,13 +191,7 @@
 		String strForward = "exitoCargarFiscalizacionAModificar";
 
 		try {
-			UsuarioDTO usuario = (UsuarioDTO) request.getSession().getAttribute(
-					Constantes.USER_LABEL_SESSION);
 			WebApplicationContext ctx = getWebApplicationContext();
-
-			IRolFachada rolFachada = (IRolFachada) ctx.getBean("rolFachada");
-			//rolFachada.verificarMenu(Constantes.MODIFICACION_FISCALIZACION_MENU, usuario.getRol());
-
 			IFiscalizacionFachada fiscalizacionFachada = (IFiscalizacionFachada) ctx
 					.getBean("fiscalizacionFachada");
 
@@ -219,19 +208,25 @@
 			long idFiscalizacion = new Long(request.getParameter("id"));
 			Fiscalizacion fiscalizacion = fiscalizacionFachada
 					.recuperarFiscalizacion(idFiscalizacion);
+			
+			List<Entidad> productores = entidadFachada.getEntidadesPorLocalidad(fiscalizacion.getPmf().getProductorForestal()
+						.getLocalidad().getId());
 
-			List<Entidad> productores = entidadFachada.getEntidadesPorLocalidad(fiscalizacion
-					.getRodal().getMarcacion().getTranzon().getPmf().getProductorForestal()
-					.getLocalidad().getId());
+			
 			List<TipoProductoDTO> tiposProducto = tipoProductoForestalFachada.recuperarTiposProductoForestalDTO();
 
 			List<PMF> pmf = ubicacionFachada.getPMFs(fiscalizacion.getProductorForestal().getId());
-			List<Tranzon> tranzones = ubicacionFachada.getTranzonesById(fiscalizacion.getRodal()
-					.getMarcacion().getTranzon().getPmf().getId());
-			List<Marcacion> marcaciones = ubicacionFachada.getMarcacionesById(fiscalizacion
-					.getRodal().getMarcacion().getTranzon().getId());
-			List<Rodal> rodales = ubicacionFachada.getRodalesById(fiscalizacion.getRodal()
-					.getMarcacion().getId());
+			
+			List<Tranzon> tranzones = ubicacionFachada.getTranzonesById(fiscalizacion.getPmf().getId());
+			if (fiscalizacion.getTranzon()!=null){
+				List<Marcacion> marcaciones = ubicacionFachada.getMarcacionesById(fiscalizacion.getTranzon().getId());
+				request.setAttribute("marcaciones", marcaciones);				
+			}
+			
+			if (fiscalizacion.getMarcacion()!=null){
+				List<Rodal> rodales = ubicacionFachada.getRodalesById(fiscalizacion.getMarcacion().getId());
+				request.setAttribute("rodales", rodales);				
+			}	
 
 			FiscalizacionDTO fiscalizacionDTO = ProviderDTO.getFiscalizacionDTO(fiscalizacion);
 			request.getSession().setAttribute("fiscalizacionDTO", fiscalizacionDTO);
@@ -245,8 +240,8 @@
 			request.setAttribute("tiposProducto", tiposProducto);
 			request.setAttribute("pmfs", pmf);
 			request.setAttribute("tranzones", tranzones);
-			request.setAttribute("marcaciones", marcaciones);
-			request.setAttribute("rodales", rodales);
+			
+			
 			request.setAttribute("periodos", periodoFachada.getPeriodosDTO());
 
 			request.setAttribute("LENIA", Constantes.LENIA_ID);
@@ -307,10 +302,6 @@
 			IFiscalizacionFachada fiscalizacionFachada = (IFiscalizacionFachada) ctx
 					.getBean("fiscalizacionFachada");
 
-			IUbicacionFachada ubicacionFachada = (IUbicacionFachada) ctx
-					.getBean("ubicacionFachada");
-			IEntidadFachada entidadFachada = (IEntidadFachada) ctx.getBean("entidadFachada");
-
 			FiscalizacionForm fiscalizacionForm = (FiscalizacionForm) form;
 			FiscalizacionDTO fiscalizacionDTO = fiscalizacionForm.getFiscalizacionDTO();
 			fiscalizacionDTO.setUsuario(usuario);
@@ -344,12 +335,7 @@
 		String strForward = "exitoRecuperarTiposDeEntidadParaFiscalizacionesAModificar";
 
 		try {
-			UsuarioDTO usuario = (UsuarioDTO) request.getSession().getAttribute(
-					Constantes.USER_LABEL_SESSION);
 			WebApplicationContext ctx = getWebApplicationContext();
-
-			IRolFachada rolFachada = (IRolFachada) ctx.getBean("rolFachada");
-
 			IEntidadFachada entidadFachada = (IEntidadFachada) ctx.getBean("entidadFachada");
 
 			String idTipoDeEntidad = request.getParameter("idTipoDeEntidad");
Index: WebContent/jsp/cargarFiscalizacionConsulta.jsp
===================================================================
--- WebContent/jsp/cargarFiscalizacionConsulta.jsp	(revision 332)
+++ WebContent/jsp/cargarFiscalizacionConsulta.jsp	(working copy)
@@ -152,7 +152,7 @@
 						</td>
 						<td>
 							<input class="botonerab" type="text" size="25" readonly="readonly"
-								   value="<c:out value='${fiscalizacion.rodal.marcacion.tranzon.pmf.nombre}- ${fiscalizacion.rodal.marcacion.tranzon.pmf.expediente}'></c:out>">																					
+								   value="<c:out value='${fiscalizacion.localizacion.nombrePmf}- ${fiscalizacion.pmf.expediente}'></c:out>">																					
 						</td>
 						<td width="25%"></td>
 					</tr>		
@@ -162,7 +162,7 @@
 						</td>
 						<td>
 							<input class="botonerab" type="text" size="25" readonly="readonly"
-								   value="<c:out value='${fiscalizacion.rodal.marcacion.tranzon.numero}- ${fiscalizacion.rodal.marcacion.tranzon.disposicion}'></c:out>">																						
+								   value="<c:out value='${fiscalizacion.tranzon.numero}- ${fiscalizacion.tranzon.disposicion}'></c:out>">																						
 						</td>
 						<td width="25%"></td>
 					</tr>	
@@ -172,7 +172,7 @@
 						</td>
 						<td>
 							<input class="botonerab" type="text" size="25" readonly="readonly"
-								   value="<c:out value='${fiscalizacion.rodal.marcacion.disposicion}'></c:out>">																					
+								   value="<c:out value='${fiscalizacion.marcacion.disposicion}'></c:out>">																					
 						</td>
 						<td width="25%"></td>
 					</tr>
Index: src/ar/com/siif/fachada/UbicacionFachada.java
===================================================================
--- src/ar/com/siif/fachada/UbicacionFachada.java	(revision 325)
+++ src/ar/com/siif/fachada/UbicacionFachada.java	(working copy)
@@ -50,6 +50,10 @@
 		}			
 	}
 
+	public Tranzon getTranzon(Long idTranzon) {
+		return this.ubicacionDAO.getTranzon(idTranzon);
+	}
+	
 	public List<Tranzon> getTranzonesById(Long idPMF) throws NegocioException {
 		try{
 			PMF pmf = ubicacionDAO.getPMF(idPMF);
Index: src/ar/com/siif/providers/ProviderDominio.java
===================================================================
--- src/ar/com/siif/providers/ProviderDominio.java	(revision 325)
+++ src/ar/com/siif/providers/ProviderDominio.java	(working copy)
@@ -23,6 +23,7 @@
 import ar.com.siif.dto.UsuarioDTO;
 import ar.com.siif.dto.ValeTransporteDTO;
 import ar.com.siif.enums.TipoDeEntidad;
+import ar.com.siif.fachada.Ubicacion;
 import ar.com.siif.negocio.Aforo;
 import ar.com.siif.negocio.BoletaDeposito;
 import ar.com.siif.negocio.CertificadoOrigen;
@@ -55,7 +56,7 @@
 public abstract class ProviderDominio {
 
 	public static Fiscalizacion getFiscalizacion(FiscalizacionDTO fiscalizacionDTO,
-			List<MuestraDTO> muestrasDTO, Rodal rodal, Entidad productorForestal,
+			List<MuestraDTO> muestrasDTO, Ubicacion ubicacion, Entidad productorForestal,
 			Entidad oficinaForestal, TipoProducto tipoProducto, Usuario usuario) {
 
 		Fiscalizacion fiscalizacion = new Fiscalizacion();
@@ -66,7 +67,7 @@
 		fiscalizacion.setOficinaAlta(oficinaForestal);
 		fiscalizacion.setPeriodoForestal(fiscalizacionDTO.getPeriodoForestal());
 		fiscalizacion.setProductorForestal(productorForestal);
-		fiscalizacion.setRodal(rodal);
+		ubicacion.setEnFiscalizacion(fiscalizacion);
 		fiscalizacion.setTamanioMuestra(fiscalizacionDTO.getTamanioMuestra());
 		fiscalizacion.setTipoProducto(tipoProducto);
 		fiscalizacion.setUsuario(usuario);
Index: WebContent/jsp/modificacionFiscalizacionProdForestales.jsp
===================================================================
--- WebContent/jsp/modificacionFiscalizacionProdForestales.jsp	(revision 325)
+++ WebContent/jsp/modificacionFiscalizacionProdForestales.jsp	(working copy)
@@ -70,13 +70,12 @@
 
 	<html:hidden property="fiscalizacionDTO.id"
 		value="${fiscalizacionDTO.id}" styleId="idFiscalizacion" />
-	
+	<html:hidden property="fiscalizacionDTO.pmf.id"
+		value="${fiscalizacionDTO.pmf.id}"/>
 	<html:hidden property="fiscalizacionDTO.productorForestal.id"
 		value="${fiscalizacionDTO.productorForestal.id}"/>
 	<html:hidden property="fiscalizacionDTO.tipoProducto.id"
 		value="${fiscalizacionDTO.tipoProducto.id}"/>		
-	<html:hidden property="fiscalizacionDTO.rodal.id"
-		value="${fiscalizacionDTO.rodal.id}"/>	
 
 	<table border="0" class="cuadrado" align="center" width="70%"
 		cellpadding="2">
@@ -237,7 +236,7 @@
 								key='SIIF.label.PlanManejoForestal' /></td>
 						<td><html:select styleId="idPMF"
 								property="idPMF" styleClass="botonerab disabled"
-								value="${fiscalizacionDTO.rodal.marcacion.tranzon.pmf.id}"
+								value="${fiscalizacionDTO.pmf.id}"
 								onchange="actualizarComboTranzon();">
 								<option value="-1">- Seleccione -</option>
 								<c:forEach items="${pmfs}" var="pmf">
@@ -253,7 +252,7 @@
 								key='SIIF.label.Tranzon' /></td>
 						<td><html:select styleId="idTranzon" property="idTranzon"
 								styleClass="botonerab disabled"
-								value="${fiscalizacionDTO.rodal.marcacion.tranzon.id}"
+								value="${fiscalizacionDTO.tranzon.id}"
 								onchange="actualizarComboMarcacion();">
 								<option value="-1">- Seleccione -</option>
 								<c:forEach items="${tranzones}" var="tranzon">
@@ -269,7 +268,7 @@
 								key='SIIF.label.Marcacion' /></td>
 						<td><html:select styleId="idMarcacion" property="idMarcacion"
 								styleClass="botonerab disabled"
-								value="${fiscalizacionDTO.rodal.marcacion.id}"
+								value="${fiscalizacionDTO.marcacion.id}"
 								onchange="actualizarComboRodal();">
 								<option value="-1">- Seleccione -</option>
 								<c:forEach items="${marcaciones}" var="marcacion">
Index: src/ar/com/siif/dao/FiscalizacionDAO.java
===================================================================
--- src/ar/com/siif/dao/FiscalizacionDAO.java	(revision 297)
+++ src/ar/com/siif/dao/FiscalizacionDAO.java	(working copy)
@@ -151,15 +151,12 @@
 		Criteria criteria = getSession().createCriteria(Fiscalizacion.class);
 
 		criteria.createAlias("productorForestal", "pf");
-		criteria.createAlias("rodal", "rod");			
 		criteria.createAlias("tipoProducto", "tp");
 		
 		criteria.add(Restrictions.conjunction()
 				.add(Restrictions.eq("productorForestal.id", idProductor))
 				.add(Restrictions.isNull("guiaForestal")));
 
-		criteria.addOrder(Order.asc("rod.id"));
-		/*criteria.addOrder(Order.asc("tp.id"));*/
 		criteria.addOrder(Order.asc("fecha"));
 
 		List<Fiscalizacion> fiscalizaciones = criteria.list();
@@ -173,6 +170,9 @@
 		fiscalizacion.setGuiaForestal(null);
 		fiscalizacion.setOficinaAlta(null);
 		fiscalizacion.setProductorForestal(null);
+		fiscalizacion.setPmf(null);
+		fiscalizacion.setMarcacion(null);
+		fiscalizacion.setTranzon(null);
 		fiscalizacion.setRodal(null);
 		fiscalizacion.setUsuario(null);
 		fiscalizacion.setTipoProducto(null);
Index: WebContent/jsp/bloqueFiscalizacionesAModificar.jsp
===================================================================
--- WebContent/jsp/bloqueFiscalizacionesAModificar.jsp	(revision 325)
+++ WebContent/jsp/bloqueFiscalizacionesAModificar.jsp	(working copy)
@@ -31,15 +31,15 @@
 							<fmt:formatDate	value='${fiscalizacion.fecha}' pattern='dd/MM/yyyy' />
 						</td>
 						<td class="botonerab">
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.pmf.expediente}"></c:out>-
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.pmf.nombre}"></c:out>
+							<c:out value="${fiscalizacion.pmf.expediente}"></c:out>-
+							<c:out value="${fiscalizacion.pmf.nombre}"></c:out>
 						</td>
 						<td class="botonerab">
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.numero}"></c:out>-
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.disposicion}"></c:out>
+							<c:out value="${fiscalizacion.tranzon.numero}"></c:out>-
+							<c:out value="${fiscalizacion.tranzon.disposicion}"></c:out>
 						</td>
 						<td class="botonerab">
-							<c:out value="${fiscalizacion.rodal.marcacion.disposicion}"></c:out>
+							<c:out value="${fiscalizacion.marcacion.disposicion}"></c:out>
 						</td>						
 						<td class="botonerab">
 							<c:out value="${fiscalizacion.rodal.nombre}"></c:out>
Index: WebContent/jsp/bloqueFiscalizacionesAAnular.jsp
===================================================================
--- WebContent/jsp/bloqueFiscalizacionesAAnular.jsp	(revision 325)
+++ WebContent/jsp/bloqueFiscalizacionesAAnular.jsp	(working copy)
@@ -77,15 +77,15 @@
 							<fmt:formatDate	value='${fiscalizacion.fecha}' pattern='dd/MM/yyyy' />
 						</td>
 						<td class="botonerab">
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.pmf.expediente}"></c:out>-
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.pmf.nombre}"></c:out>
+							<c:out value="${fiscalizacion.pmf.expediente}"></c:out>-
+							<c:out value="${fiscalizacion.pmf.nombre}"></c:out>
 						</td>
 						<td class="botonerab">
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.numero}"></c:out>-
-							<c:out value="${fiscalizacion.rodal.marcacion.tranzon.disposicion}"></c:out>
+							<c:out value="${fiscalizacion.tranzon.numero}"></c:out>-
+							<c:out value="${fiscalizacion.tranzon.disposicion}"></c:out>
 						</td>
 						<td class="botonerab">
-							<c:out value="${fiscalizacion.rodal.marcacion.disposicion}"></c:out>
+							<c:out value="${fiscalizacion.marcacion.disposicion}"></c:out>
 						</td>						
 						<td class="botonerab">
 							<c:out value="${fiscalizacion.rodal.nombre}"></c:out>
Index: src/ar/com/siif/fachada/IUbicacionFachada.java
===================================================================
--- src/ar/com/siif/fachada/IUbicacionFachada.java	(revision 325)
+++ src/ar/com/siif/fachada/IUbicacionFachada.java	(working copy)
@@ -20,6 +20,8 @@
 
 	public List<Tranzon> getTranzonesById(Long idPMF) throws NegocioException;
 
+	public Tranzon getTranzon(Long idTranzon);	
+	
 	public List<Rodal> getRodales();
 	
 	public List<Rodal> getRodalesById(Long idMarcacion) throws NegocioException;	
Index: src/ar/com/siif/struts/actions/ConsultasFiscalizacionAction.java
===================================================================
--- src/ar/com/siif/struts/actions/ConsultasFiscalizacionAction.java	(revision 297)
+++ src/ar/com/siif/struts/actions/ConsultasFiscalizacionAction.java	(working copy)
@@ -176,10 +176,8 @@
 		try {
 
 			String paramForward = request.getParameter("paramForward");			
-			UsuarioDTO usuario = (UsuarioDTO)request.getSession().getAttribute(Constantes.USER_LABEL_SESSION);			
-			
 			WebApplicationContext ctx = getWebApplicationContext();			
-			IRolFachada rolFachada = (IRolFachada) ctx.getBean("rolFachada");
+
 			
 			/*if(paramForward.equals(Constantes.METODO_RECUPERAR_FISCALIZACIONES_CON_GUIA_FORESTAL)){
 				rolFachada.verificarMenu(Constantes.CONSULTA_FISCALIZACIONES_CON_GUIA_MENU,usuario.getRol());											
Index: WebContent/jsp/altaFiscalizacionProdForestales.jsp
===================================================================
--- WebContent/jsp/altaFiscalizacionProdForestales.jsp	(revision 325)
+++ WebContent/jsp/altaFiscalizacionProdForestales.jsp	(working copy)
@@ -209,7 +209,7 @@
 						</td>
 						<td width="4%"></td>						
 						<td align="left">
-							<select id="idPMF" class="botonerab" name="fiscalizacionDTO.idPlanManejoForestal" disabled="disabled" 
+							<select id="idPMF" class="botonerab" name="fiscalizacionDTO.pmf.id" disabled="disabled" 
 									onchange="actualizarComboTranzon();">
 								<option value="-1">- Seleccione -</option>						
 							</select>					
@@ -221,7 +221,7 @@
 						</td>
 						<td width="4%"></td>						
 						<td align="left">
-							<select id="idTranzon" class="botonerab" name="fiscalizacionDTO.idTranzon" disabled="disabled" 
+							<select id="idTranzon" class="botonerab" name="fiscalizacionDTO.tranzon.id" disabled="disabled" 
 									onchange="actualizarComboMarcacion();">
 								<option value="-1">- Seleccione -</option>
 							</select>					
@@ -233,7 +233,7 @@
 						</td>
 						<td width="4%"></td>
 						<td align="left">
-							<select id="idMarcacion" class="botonerab" name="fiscalizacionDTO.idMarcacion" disabled="disabled" 
+							<select id="idMarcacion" class="botonerab" name="fiscalizacionDTO.marcacion.id" disabled="disabled" 
 									onchange="actualizarComboRodal();">
 								<option value="-1">- Seleccione -</option>
 							</select>					
Index: src/ar/com/siif/dto/FiscalizacionDTO.java
===================================================================
--- src/ar/com/siif/dto/FiscalizacionDTO.java	(revision 285)
+++ src/ar/com/siif/dto/FiscalizacionDTO.java	(working copy)
@@ -22,15 +22,39 @@
 	//private long idTipoProductoForestal;
 	private TipoProductoDTO tipoProducto;	
 
-	private long idPlanManejoForestal;
+	//private long idRodal;
+	private RodalDTO rodal;
+
+	private MarcacionDTO marcacion;
 	
-	private long idTranzon;
+	private TranzonDTO tranzon;
 	
-	private long idMarcacion;
+	private PMFDTO pmf;
 	
-	//private long idRodal;
-	private RodalDTO rodal;
+	public MarcacionDTO getMarcacion() {
+		return marcacion;
+	}
 
+	public void setMarcacion(MarcacionDTO marcacion) {
+		this.marcacion = marcacion;
+	}
+
+	public TranzonDTO getTranzon() {
+		return tranzon;
+	}
+
+	public void setTranzon(TranzonDTO tranzon) {
+		this.tranzon = tranzon;
+	}
+
+	public PMFDTO getPmf() {
+		return pmf;
+	}
+
+	public void setPmf(PMFDTO pmf) {
+		this.pmf = pmf;
+	}
+
 	//private long idOficinaForestal;	
 	private EntidadDTO oficinaAlta;
 	
@@ -46,6 +70,9 @@
 		productorForestal = new EntidadDTO();
 		tipoProducto = new TipoProductoDTO();
 		rodal = new RodalDTO();
+		tranzon = new TranzonDTO();
+		marcacion = new MarcacionDTO();
+		pmf = new PMFDTO();
 		oficinaAlta = new EntidadDTO();
 		usuario = new UsuarioDTO();
 	}
@@ -98,30 +125,7 @@
 		this.tamanioMuestra = tamanioMuestra;
 	}
 
-	public long getIdPlanManejoForestal() {
-		return idPlanManejoForestal;
-	}
 
-	public void setIdPlanManejoForestal(long idPlanManejoForestal) {
-		this.idPlanManejoForestal = idPlanManejoForestal;
-	}
-
-	public long getIdTranzon() {
-		return idTranzon;
-	}
-
-	public void setIdTranzon(long idTranzon) {
-		this.idTranzon = idTranzon;
-	}
-
-	public long getIdMarcacion() {
-		return idMarcacion;
-	}
-
-	public void setIdMarcacion(long idMarcacion) {
-		this.idMarcacion = idMarcacion;
-	}
-
 	public List<MuestraDTO> getMuestra() {
 		return muestra;
 	}
Index: src/ar/com/siif/providers/ProviderDTO.java
===================================================================
--- src/ar/com/siif/providers/ProviderDTO.java	(revision 290)
+++ src/ar/com/siif/providers/ProviderDTO.java	(working copy)
@@ -98,34 +98,34 @@
 	public static TranzonDTO getTranzonDTO(Tranzon tranzon) {
 
 		TranzonDTO tranzonDTO = new TranzonDTO();
-
-		tranzonDTO.setId(tranzon.getId());
-		tranzonDTO.setDisposicion(tranzon.getDisposicion());
-		tranzonDTO.setNumero(tranzon.getNumero());
-		tranzonDTO.setPmf(ProviderDTO.getPMFDTO(tranzon.getPmf()));
-
+		if (tranzon != null) {
+			tranzonDTO.setId(tranzon.getId());
+			tranzonDTO.setDisposicion(tranzon.getDisposicion());
+			tranzonDTO.setNumero(tranzon.getNumero());
+			tranzonDTO.setPmf(ProviderDTO.getPMFDTO(tranzon.getPmf()));
+		}
 		return tranzonDTO;
 	}
 
 	public static MarcacionDTO getMarcacionDTO(Marcacion marcacion) {
 
 		MarcacionDTO marcacionDTO = new MarcacionDTO();
-
-		marcacionDTO.setId(marcacion.getId());
-		marcacionDTO.setDisposicion(marcacion.getDisposicion());
-		marcacionDTO.setTranzon(ProviderDTO.getTranzonDTO(marcacion.getTranzon()));
-
+		if (marcacion != null) {
+			marcacionDTO.setId(marcacion.getId());
+			marcacionDTO.setDisposicion(marcacion.getDisposicion());
+			marcacionDTO.setTranzon(ProviderDTO.getTranzonDTO(marcacion.getTranzon()));
+		}
 		return marcacionDTO;
 	}
 
 	public static RodalDTO getRodalDTO(Rodal rodal) {
 
 		RodalDTO rodalDTO = new RodalDTO();
-
-		rodalDTO.setId(rodal.getId());
-		rodalDTO.setNombre(rodal.getNombre());
-		rodalDTO.setMarcacion(ProviderDTO.getMarcacionDTO(rodal.getMarcacion()));
-
+		if (rodal != null) {
+			rodalDTO.setId(rodal.getId());
+			rodalDTO.setNombre(rodal.getNombre());
+			rodalDTO.setMarcacion(ProviderDTO.getMarcacionDTO(rodal.getMarcacion()));
+		}
 		return rodalDTO;
 	}
 
@@ -220,7 +220,7 @@
 		}
 	}
 
-	//Genera una FiscalizacionDTO sin los datos de la guiaForestal.
+	// Genera una FiscalizacionDTO sin los datos de la guiaForestal.
 	private static FiscalizacionDTO getFiscalizacionDTODatosGenerales(Fiscalizacion fiscalizacion) {
 
 		List<MuestraDTO> listaMuestrasDTO = new ArrayList<MuestraDTO>();
@@ -228,15 +228,15 @@
 
 		fiscalizacionDTO.setCantidadMts(fiscalizacion.getCantidadMts());
 		fiscalizacionDTO.setCantidadUnidades(fiscalizacion.getCantidadUnidades());
-		fiscalizacionDTO.setFecha(Fecha.getFechaDDMMAAAASlash(Fecha
-				.dateToStringDDMMAAAA(fiscalizacion.getFecha())));
+		fiscalizacionDTO.setFecha(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(fiscalizacion.getFecha())));
 		fiscalizacionDTO.setId(fiscalizacion.getId());
 		fiscalizacionDTO.setOficinaAlta(ProviderDTO.getEntidadDTO(fiscalizacion.getOficinaAlta()));
-		fiscalizacionDTO.setProductorForestal(ProviderDTO.getEntidadDTO(fiscalizacion
-				.getProductorForestal()));
+		fiscalizacionDTO.setProductorForestal(ProviderDTO.getEntidadDTO(fiscalizacion.getProductorForestal()));
 		fiscalizacionDTO.setRodal(ProviderDTO.getRodalDTO(fiscalizacion.getRodal()));
-		fiscalizacionDTO.setTipoProducto(ProviderDTO.getTipoProductoDTO(fiscalizacion
-				.getTipoProducto()));
+		fiscalizacionDTO.setMarcacion(ProviderDTO.getMarcacionDTO(fiscalizacion.getMarcacion()));
+		fiscalizacionDTO.setTranzon(ProviderDTO.getTranzonDTO(fiscalizacion.getTranzon()));
+		fiscalizacionDTO.setPmf(ProviderDTO.getPMFDTO(fiscalizacion.getPmf()));
+		fiscalizacionDTO.setTipoProducto(ProviderDTO.getTipoProductoDTO(fiscalizacion.getTipoProducto()));
 		fiscalizacionDTO.setUsuario(ProviderDTO.getUsuarioDTO(fiscalizacion.getUsuario()));
 		fiscalizacionDTO.setPeriodoForestal(fiscalizacion.getPeriodoForestal());
 		fiscalizacionDTO.setTamanioMuestra(fiscalizacion.getTamanioMuestra());
@@ -255,8 +255,7 @@
 		FiscalizacionDTO fiscalizacionDTO = getFiscalizacionDTODatosGenerales(fiscalizacion);
 
 		if (fiscalizacion.getGuiaForestal() != null) {
-			fiscalizacionDTO.setGuiaForestal(ProviderDTO
-					.getGuiaForestalDTODatosGenerales(fiscalizacion.getGuiaForestal()));
+			fiscalizacionDTO.setGuiaForestal(ProviderDTO.getGuiaForestalDTODatosGenerales(fiscalizacion.getGuiaForestal()));
 		}
 
 		return fiscalizacionDTO;
@@ -274,7 +273,7 @@
 		return muestraDTO;
 	}
 
-	//Genera una GuiaForestalDTO sin los datos de la Fiscalización.
+	// Genera una GuiaForestalDTO sin los datos de la Fiscalización.
 	private static GuiaForestalDTO getGuiaForestalDTODatosGenerales(GuiaForestal guiaForestal) {
 
 		GuiaForestalDTO guiaForestalDTO = new GuiaForestalDTO();
@@ -284,10 +283,8 @@
 
 		guiaForestalDTO.setId(guiaForestal.getId());
 		guiaForestalDTO.setDistanciaAforoMovil(guiaForestal.getDistanciaAforoMovil());
-		guiaForestalDTO.setFecha(Fecha.getFechaDDMMAAAASlash(Fecha
-				.dateToStringDDMMAAAA(guiaForestal.getFecha())));
-		guiaForestalDTO.setFechaVencimiento(Fecha.getFechaDDMMAAAASlash(Fecha
-				.dateToStringDDMMAAAA(guiaForestal.getFechaVencimiento())));
+		guiaForestalDTO.setFecha(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(guiaForestal.getFecha())));
+		guiaForestalDTO.setFechaVencimiento(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(guiaForestal.getFechaVencimiento())));
 		guiaForestalDTO.setImporteTotal(guiaForestal.getImporteTotal());
 		guiaForestalDTO.setInspFiscalizacion(guiaForestal.getInspFiscalizacion());
 		guiaForestalDTO.setLocalidad(ProviderDTO.getLocalidadDTO(guiaForestal.getLocalidad()));
@@ -295,8 +292,7 @@
 		guiaForestalDTO.setObservaciones(guiaForestal.getObservaciones());
 
 		guiaForestalDTO.setPeriodoForestal(guiaForestal.getPeriodoForestal());
-		guiaForestalDTO.setProductorForestal(ProviderDTO.getEntidadDTO(guiaForestal
-				.getProductorForestal()));
+		guiaForestalDTO.setProductorForestal(ProviderDTO.getEntidadDTO(guiaForestal.getProductorForestal()));
 
 		if (guiaForestal.getRodal() != null) {
 			guiaForestalDTO.setRodal(ProviderDTO.getRodalDTO(guiaForestal.getRodal()));
@@ -314,8 +310,7 @@
 
 		List<ValeTransporte> listaValesTransporte = guiaForestal.getValesTransporte();
 		for (ValeTransporte valeTransporte : listaValesTransporte) {
-			listaValesTransporteDTO.add(ProviderDTO.getValeTransporteDTO(valeTransporte,
-					guiaForestalDTO));
+			listaValesTransporteDTO.add(ProviderDTO.getValeTransporteDTO(valeTransporte, guiaForestalDTO));
 		}
 		guiaForestalDTO.setValesTransporte(listaValesTransporteDTO);
 
@@ -335,17 +330,15 @@
 		GuiaForestalDTO guiaForestalDTO = getGuiaForestalDTODatosGenerales(guiaForestal);
 
 		for (Fiscalizacion fiscalizacion : guiaForestal.getFiscalizaciones()) {
-			guiaForestalDTO.getFiscalizaciones().add(
-					ProviderDTO.getFiscalizacionDTODatosGenerales(fiscalizacion));
+			guiaForestalDTO.getFiscalizaciones().add(ProviderDTO.getFiscalizacionDTODatosGenerales(fiscalizacion));
 		}
 
-		//guiaForestalDTO.setFiscalizacion(ProviderDTO.getFiscalizacionDTODatosGenerales(guiaForestal.getFiscalizacion()));
+		// guiaForestalDTO.setFiscalizacion(ProviderDTO.getFiscalizacionDTODatosGenerales(guiaForestal.getFiscalizacion()));
 
 		return guiaForestalDTO;
 	}
 
-	public static BoletaDepositoDTO getBoletaDepositoDTO(BoletaDeposito boleta,
-			GuiaForestalDTO guiaForestalDTO) {
+	public static BoletaDepositoDTO getBoletaDepositoDTO(BoletaDeposito boleta, GuiaForestalDTO guiaForestalDTO) {
 
 		BoletaDepositoDTO boletaDTO = new BoletaDepositoDTO();
 
@@ -353,11 +346,9 @@
 		boletaDTO.setConcepto(boleta.getConcepto());
 		boletaDTO.setEfectivoCheque(boleta.getEfectivoCheque());
 		if (boleta.getFechaPago() != null) {
-			boletaDTO.setFechaPago(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(boleta
-					.getFechaPago())));
+			boletaDTO.setFechaPago(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(boleta.getFechaPago())));
 		}
-		boletaDTO.setFechaVencimiento(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(boleta
-				.getFechaVencimiento())));
+		boletaDTO.setFechaVencimiento(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(boleta.getFechaVencimiento())));
 		boletaDTO.setGuiaForestal(guiaForestalDTO);
 		boletaDTO.setIdBoleta(boleta.getId());
 		boletaDTO.setMonto(boleta.getMonto());
@@ -366,8 +357,7 @@
 		return boletaDTO;
 	}
 
-	public static ValeTransporteDTO getValeTransporteDTO(ValeTransporte vale,
-			GuiaForestalDTO guiaForestalDTO) {
+	public static ValeTransporteDTO getValeTransporteDTO(ValeTransporte vale, GuiaForestalDTO guiaForestalDTO) {
 
 		ValeTransporteDTO valeDTO = new ValeTransporteDTO();
 
@@ -377,11 +367,9 @@
 		valeDTO.setDominio(vale.getDominio());
 		valeDTO.setEspecie(vale.getEspecie());
 		if (vale.getFechaDevolucion() != null) {
-			valeDTO.setFechaDevolucion(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(vale
-					.getFechaDevolucion())));
+			valeDTO.setFechaDevolucion(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(vale.getFechaDevolucion())));
 		}
-		valeDTO.setFechaVencimiento(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(vale
-				.getFechaVencimiento())));
+		valeDTO.setFechaVencimiento(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(vale.getFechaVencimiento())));
 		valeDTO.setGuiaForestal(guiaForestalDTO);
 		valeDTO.setMarca(vale.getMarca());
 		valeDTO.setNroPiezas(vale.getNroPiezas());
@@ -403,7 +391,7 @@
 		subImporteDTO.setEstado(subImporte.getEstado());
 		subImporteDTO.setGuiaForestal(guiaDTO);
 		subImporteDTO.setId(subImporte.getId());
-		//subImporteDTO.setImporte(subImporte.getImporte());
+		// subImporteDTO.setImporte(subImporte.getImporte());
 		subImporteDTO.setImporte(MathUtils.round(subImporte.getImporte(), 2));
 		subImporteDTO.setTipoProducto(ProviderDTO.getTipoProductoDTO(subImporte.getTipoProducto()));
 		subImporteDTO.setValorAforos(subImporte.getValorAforos());
@@ -427,21 +415,19 @@
 
 		localidadDTO.setId(localidad.getId());
 		localidadDTO.setNombre(localidad.getNombre());
-		localidadDTO.setProvinciaDestinoDTO(ProviderDTO.getProvinciaDestinoDTO(localidad
-				.getProvinciaDestino()));
+		localidadDTO.setProvinciaDestinoDTO(ProviderDTO.getProvinciaDestinoDTO(localidad.getProvinciaDestino()));
 
 		return localidadDTO;
 	}
 
-	public static TipoProductoEnCertificadoDTO getTipoProductoEnCertificadoDTO(
-			TipoProductoEnCertificado tipoProducto, CertificadoOrigenDTO certificadoDTO) {
+	public static TipoProductoEnCertificadoDTO getTipoProductoEnCertificadoDTO(TipoProductoEnCertificado tipoProducto,
+			CertificadoOrigenDTO certificadoDTO) {
 
 		TipoProductoEnCertificadoDTO tipoProductoDTO = new TipoProductoEnCertificadoDTO();
 
 		tipoProductoDTO.setCertificadoOrigen(certificadoDTO);
 		tipoProductoDTO.setId(tipoProducto.getId());
-		tipoProductoDTO.setTipoProductoExportacion(ProviderDTO.getTipoProductoDTO(tipoProducto
-				.getTipoProductoExportacion()));
+		tipoProductoDTO.setTipoProductoExportacion(ProviderDTO.getTipoProductoDTO(tipoProducto.getTipoProductoExportacion()));
 		tipoProductoDTO.setVolumenTipoProducto(tipoProducto.getVolumenTipoProducto());
 
 		return tipoProductoDTO;
@@ -452,11 +438,9 @@
 		CertificadoOrigenDTO certificadoDTO = new CertificadoOrigenDTO();
 
 		certificadoDTO.setExportador(ProviderDTO.getEntidadDTO(certificado.getExportador()));
-		certificadoDTO.setFecha(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(certificado
-				.getFecha())));
+		certificadoDTO.setFecha(Fecha.getFechaDDMMAAAASlash(Fecha.dateToStringDDMMAAAA(certificado.getFecha())));
 		certificadoDTO.setId(certificado.getId());
-		certificadoDTO.setLocalidadDestino(ProviderDTO.getLocalidadDestinoDTO(certificado
-				.getLocalidadDestino()));
+		certificadoDTO.setLocalidadDestino(ProviderDTO.getLocalidadDestinoDTO(certificado.getLocalidadDestino()));
 		certificadoDTO.setNroCertificado(certificado.getNroCertificado());
 		certificadoDTO.setNroFactura(certificado.getNroFactura());
 		certificadoDTO.setNroRemito(certificado.getNroRemito());
@@ -471,8 +455,7 @@
 
 		List<TipoProductoEnCertificadoDTO> listaTipoProdDTO = new ArrayList<TipoProductoEnCertificadoDTO>();
 		for (TipoProductoEnCertificado tipoProd : certificado.getTiposProductoEnCertificado()) {
-			listaTipoProdDTO.add(ProviderDTO.getTipoProductoEnCertificadoDTO(tipoProd,
-					certificadoDTO));
+			listaTipoProdDTO.add(ProviderDTO.getTipoProductoEnCertificadoDTO(tipoProd, certificadoDTO));
 		}
 
 		certificadoDTO.setTiposProductoEnCertificado(listaTipoProdDTO);
@@ -480,248 +463,243 @@
 		return certificadoDTO;
 	}
 
-	/*	
-		public static ActaMartilladoDTO getActaMartilladoDTO(FiscalizacionForm form){
-			
-			ActaMartilladoDTO acta = new ActaMartilladoDTO();
-			acta.setNroOrden(form.getNroOrden());
-			acta.setProductorForestal(form.getProductorForestal());
-			acta.setPeriodoForestal(form.getPeriodoForestal());
-			acta.setExpediente(form.getExpediente());
-			acta.setFecha(Fecha.stringDDMMAAAAToDate(form.getFecha()));
-			acta.setUbicacionZonal(form.getUbicacionZonal());
-			acta.setCantidadUnidades(form.getCantidadUnidades());
-			acta.setTipoTransporte(form.getTipoTransporte());
-			acta.setCantidadMts(form.getCantidadMts());
-			acta.setTamanio(form.getTamanio());
-			if(form.getGuiaForestal() != null && form.getGuiaForestal().getNroGuia() != 0){
-				acta.setGuiaForestal(form.getGuiaForestal());
-			}	
-			
-			return acta;
-		}
-
-		public static ActaMartilladoDTO getActaMartilladoDTO(ActaMartillado acta){
-			
-			ActaMartilladoDTO actaDTO = new ActaMartilladoDTO();
-			actaDTO.setNroOrden(acta.getNroOrden());
-			actaDTO.setProductorForestal(getEntidadDTO(acta.getProductorForestal()));
-			actaDTO.setPeriodoForestal(acta.getPeriodoForestal());
-			actaDTO.setExpediente(acta.getExpediente());
-			actaDTO.setFecha(acta.getFecha());
-			actaDTO.setUbicacionZonal(acta.getUbicacionZonal());
-			actaDTO.setCantidadUnidades(acta.getCantidadUnidades());
-			actaDTO.setTipoTransporte(acta.getTipoTransporte());
-			actaDTO.setCantidadMts(acta.getCantidadMts());
-			actaDTO.setTamanio(acta.getTamanio());		
-			if(acta.getGuiaForestal() != null && acta.getGuiaForestal().getNroGuia() != 0){
-				actaDTO.setGuiaForestal(getGuiaForestalDTO(acta.getGuiaForestal(),actaDTO));
-			}
-			
-			return actaDTO;
-		}	
-
-		public static ActaMartilladoDTO getActaMartilladoDTO(ActaMartillado acta, GuiaForestalDTO guiaDTO){
-			
-			ActaMartilladoDTO actaDTO = new ActaMartilladoDTO();
-			actaDTO.setNroOrden(acta.getNroOrden());
-			actaDTO.setProductorForestal(getEntidadDTO(acta.getProductorForestal()));
-			actaDTO.setPeriodoForestal(acta.getPeriodoForestal());
-			actaDTO.setExpediente(acta.getExpediente());
-			actaDTO.setFecha(acta.getFecha());
-			actaDTO.setUbicacionZonal(acta.getUbicacionZonal());
-			actaDTO.setCantidadUnidades(acta.getCantidadUnidades());
-			actaDTO.setTipoTransporte(acta.getTipoTransporte());
-			actaDTO.setCantidadMts(acta.getCantidadMts());
-			actaDTO.setTamanio(acta.getTamanio());		
-			actaDTO.setGuiaForestal(guiaDTO);
-			
-			return actaDTO;
-		}	
-		
-		public static GuiaForestalDTO getGuiaForestalDTO(GuiaForestalForm form, Usuario usuario){
-							
-			GuiaForestalDTO guia = new GuiaForestalDTO();
-			
-			guia.setNroGuia(form.getNroGuia());
-			guia.setFiscalizacion(form.getFiscalizacion());
-			guia.setLugarCorte(form.getLugarCorte());
-			//guia.setPermisionario(form.getPermisionario());
-			guia.setFechaVencimiento(Fecha.stringDDMMAAAAToDate(form.getFechaVencimiento()));
-			guia.setDistanciaAforoMovil(form.getDistanciaAforoMovil());
-			guia.setNroExpediente(form.getNroExpediente());
-			guia.setPeriodoForestal(form.getPeriodoForestal());
-			guia.setDisposicionNro(form.getDisposicionNro());
-			guia.setTipo(form.getTipo());
-			guia.setEstado(form.getEstado());
-			guia.setEspecie(form.getEspecie());
-			guia.setVolumenTotal(form.getVolumenTotal());
-			guia.setCantidad(form.getCantidad());
-			guia.setImporte(form.getImporte());
-			guia.setAforo(form.getAforo());
-			guia.setInspFiscalizacion(form.getInspFiscalizacion());
-			guia.setValorAforos(form.getValorAforos());
-			guia.setObservaciones(form.getObservaciones());
-			guia.setLocalidad(form.getLocalidad());
-			guia.setFecha(Fecha.stringDDMMAAAAToDate(form.getFecha()));		
-			guia.setValesTransporte(normalizarValesTransporte(form.getValesTransporte()));
-			guia.setBoletasDeposito(normalizarBoletasDeposito(form.getBoletasDeposito()));
-			guia.setUsuario(usuario);
-			
-			return guia;
-		}
-
-		public static GuiaForestalDTO getGuiaForestalDTO(GuiaForestal guia, ActaMartilladoDTO actaDTO){
-			
-			GuiaForestalDTO guiaDTO = new GuiaForestalDTO();
-			
-			guiaDTO.setNroGuia(guia.getNroGuia());
-			guiaDTO.setFiscalizacion(actaDTO);
-			guiaDTO.setLugarCorte(guia.getLugarCorte());
-			//guiaDTO.setPermisionario(guia.getPermisionario().getId());
-			guiaDTO.setFechaVencimiento(guia.getFechaVencimiento());
-			guiaDTO.setDistanciaAforoMovil(guia.getDistanciaAforoMovil());
-			guiaDTO.setNroExpediente(guia.getNroExpediente());
-			guiaDTO.setPeriodoForestal(guia.getPeriodoForestal());
-			guiaDTO.setDisposicionNro(guia.getDisposicionNro());
-			guiaDTO.setTipo(guia.getTipo());
-			guiaDTO.setEstado(guia.getEstado());
-			guiaDTO.setEspecie(guia.getEspecie());
-			guiaDTO.setVolumenTotal(guia.getVolumenTotal());
-			guiaDTO.setCantidad(guia.getCantidad());
-			guiaDTO.setImporte(guia.getImporte());
-			guiaDTO.setAforo(guia.getAforo());
-			guiaDTO.setInspFiscalizacion(guia.getInspFiscalizacion());
-			guiaDTO.setValorAforos(guia.getValorAforos());
-			guiaDTO.setObservaciones(guia.getObservaciones());
-			guiaDTO.setLocalidad(guia.getLocalidad());
-			guiaDTO.setFecha(guia.getFecha());		
-			guiaDTO.setValesTransporte(getValesTransporteDTO(guia.getValesTransporte(),guiaDTO));
-			guiaDTO.setBoletasDeposito(getBoletasDesposito(guia.getBoletasDeposito(),guiaDTO));
-			guiaDTO.setUsuario(guia.getUsuario());
-			
-			return guiaDTO;		
-		}	
-		
-		public static GuiaForestalDTO getGuiaForestalDTO(GuiaForestal guia){
-			
-			GuiaForestalDTO guiaDTO = new GuiaForestalDTO();
-			
-			guiaDTO.setNroGuia(guia.getNroGuia());
-			guiaDTO.setFiscalizacion(getActaMartilladoDTO(guia.getActaMartillado(),guiaDTO));
-			guiaDTO.setLugarCorte(guia.getLugarCorte());
-			//guiaDTO.setPermisionario(guia.getPermisionario().getId());
-			guiaDTO.setFechaVencimiento(guia.getFechaVencimiento());
-			guiaDTO.setDistanciaAforoMovil(guia.getDistanciaAforoMovil());
-			guiaDTO.setNroExpediente(guia.getNroExpediente());
-			guiaDTO.setPeriodoForestal(guia.getPeriodoForestal());
-			guiaDTO.setDisposicionNro(guia.getDisposicionNro());
-			guiaDTO.setTipo(guia.getTipo());
-			guiaDTO.setEstado(guia.getEstado());
-			guiaDTO.setEspecie(guia.getEspecie());
-			guiaDTO.setVolumenTotal(guia.getVolumenTotal());
-			guiaDTO.setCantidad(guia.getCantidad());
-			guiaDTO.setImporte(guia.getImporte());
-			guiaDTO.setAforo(guia.getAforo());
-			guiaDTO.setInspFiscalizacion(guia.getInspFiscalizacion());
-			guiaDTO.setValorAforos(guia.getValorAforos());
-			guiaDTO.setObservaciones(guia.getObservaciones());
-			guiaDTO.setLocalidad(guia.getLocalidad());
-			guiaDTO.setFecha(guia.getFecha());		
-			guiaDTO.setValesTransporte(getValesTransporteDTO(guia.getValesTransporte(),guiaDTO));
-			guiaDTO.setBoletasDeposito(getBoletasDesposito(guia.getBoletasDeposito(),guiaDTO));
-			guiaDTO.setUsuario(guia.getUsuario());
-			
-			return guiaDTO;		
-		}	
-		
-		public static List<BoletaDepositoDTO> getBoletasDesposito(List<BoletaDeposito> boletas, GuiaForestalDTO guiaDTO){
-			
-			List<BoletaDepositoDTO> boletasDTO = new ArrayList<BoletaDepositoDTO>();
-			
-			for (BoletaDeposito boleta : boletas) {
-				
-				boletasDTO.add(getBoletaDesposito(boleta,guiaDTO));
-			}
-			
-			return boletasDTO;
-		}	
-		
-		public static BoletaDepositoDTO getBoletaDesposito(BoletaDeposito boleta, GuiaForestalDTO guiaDTO){
-			
-			BoletaDepositoDTO boletaDTO = new BoletaDepositoDTO();
-			
-			boletaDTO.setArea(boleta.getArea());
-			boletaDTO.setConcepto(boleta.getConcepto());
-			boletaDTO.setEfectivoCheque(boleta.getEfectivoCheque());
-			boletaDTO.setFechaPago(Fecha.dateToStringDDMMAAAA(boleta.getFechaPago()));
-			boletaDTO.setFechaVencimiento(Fecha.dateToStringDDMMAAAA(boleta.getFechaVencimiento()));
-			boletaDTO.setMonto(boleta.getMonto());
-			boletaDTO.setNumero(boleta.getNumero());
-			boletaDTO.setGuiaForestal(guiaDTO);
-			
-			return boletaDTO;	
-		}
-		
-		public static List<ValeTransporteDTO> getValesTransporteDTO(List<ValeTransporte> vales, GuiaForestalDTO guiaDTO){
-			
-			List<ValeTransporteDTO> valesDTO = new ArrayList<ValeTransporteDTO>();
-			
-			for (ValeTransporte vale : vales) {
-				
-				valesDTO.add(getValeTransporte(vale,guiaDTO));
-			}
-			
-			return valesDTO;
-		}
-		
-		public static ValeTransporteDTO getValeTransporte(ValeTransporte vale, GuiaForestalDTO guiaDTO){
-			
-			ValeTransporteDTO valeDTO = new ValeTransporteDTO();
-			
-			valeDTO.setCantidadMts(vale.getCantidadMts());
-			valeDTO.setDestino(vale.getDestino());
-			valeDTO.setDominio(vale.getDominio());
-			valeDTO.setEspecie(vale.getEspecie());
-			valeDTO.setFecha(Fecha.dateToStringDDMMAAAA(vale.getFecha()));
-			valeDTO.setFechaVencimiento(Fecha.dateToStringDDMMAAAA(vale.getFechaVencimiento()));
-			valeDTO.setMarca(vale.getMarca());
-			valeDTO.setNroPiezas(vale.getNroPiezas());
-			valeDTO.setNumero(vale.getNumero());
-			valeDTO.setOrigen(vale.getOrigen());
-			valeDTO.setProducto(vale.getProducto());
-			valeDTO.setGuiaForestal(guiaDTO);
-			valeDTO.setVehiculo(vale.getVehiculo());
-			
-			return valeDTO;
-		}
-		
-		private static List<BoletaDepositoDTO> normalizarBoletasDeposito(List<BoletaDepositoDTO> boletasDeposito) 
-		{
-			List<BoletaDepositoDTO> boletasAEliminar = new ArrayList<BoletaDepositoDTO>();
-			for (BoletaDepositoDTO boleta : boletasDeposito) {
-				
-				if(boleta.getNumero() == 0){
-					boletasAEliminar.add(boleta);
-				}
-			}
-			boletasDeposito.removeAll(boletasAEliminar);
-			
-			return boletasDeposito;
-		}
-
-		private static List<ValeTransporteDTO> normalizarValesTransporte(List<ValeTransporteDTO> valesTransporte) 
-		{
-
-			List<ValeTransporteDTO> valesAEliminar = new ArrayList<ValeTransporteDTO>();
-			for (ValeTransporteDTO vale : valesTransporte) {
-				
-				if(vale.getNumero() == 0){
-					valesAEliminar.add(vale);
-				}
-			}
-			valesTransporte.removeAll(valesAEliminar);
-			
-			return valesTransporte;
-		}*/
+	/*
+	 * public static ActaMartilladoDTO getActaMartilladoDTO(FiscalizacionForm
+	 * form){
+	 * 
+	 * ActaMartilladoDTO acta = new ActaMartilladoDTO();
+	 * acta.setNroOrden(form.getNroOrden());
+	 * acta.setProductorForestal(form.getProductorForestal());
+	 * acta.setPeriodoForestal(form.getPeriodoForestal());
+	 * acta.setExpediente(form.getExpediente());
+	 * acta.setFecha(Fecha.stringDDMMAAAAToDate(form.getFecha()));
+	 * acta.setUbicacionZonal(form.getUbicacionZonal());
+	 * acta.setCantidadUnidades(form.getCantidadUnidades());
+	 * acta.setTipoTransporte(form.getTipoTransporte());
+	 * acta.setCantidadMts(form.getCantidadMts());
+	 * acta.setTamanio(form.getTamanio()); if(form.getGuiaForestal() != null &&
+	 * form.getGuiaForestal().getNroGuia() != 0){
+	 * acta.setGuiaForestal(form.getGuiaForestal()); }
+	 * 
+	 * return acta; }
+	 * 
+	 * public static ActaMartilladoDTO getActaMartilladoDTO(ActaMartillado
+	 * acta){
+	 * 
+	 * ActaMartilladoDTO actaDTO = new ActaMartilladoDTO();
+	 * actaDTO.setNroOrden(acta.getNroOrden());
+	 * actaDTO.setProductorForestal(getEntidadDTO(acta.getProductorForestal()));
+	 * actaDTO.setPeriodoForestal(acta.getPeriodoForestal());
+	 * actaDTO.setExpediente(acta.getExpediente());
+	 * actaDTO.setFecha(acta.getFecha());
+	 * actaDTO.setUbicacionZonal(acta.getUbicacionZonal());
+	 * actaDTO.setCantidadUnidades(acta.getCantidadUnidades());
+	 * actaDTO.setTipoTransporte(acta.getTipoTransporte());
+	 * actaDTO.setCantidadMts(acta.getCantidadMts());
+	 * actaDTO.setTamanio(acta.getTamanio()); if(acta.getGuiaForestal() != null
+	 * && acta.getGuiaForestal().getNroGuia() != 0){
+	 * actaDTO.setGuiaForestal(getGuiaForestalDTO
+	 * (acta.getGuiaForestal(),actaDTO)); }
+	 * 
+	 * return actaDTO; }
+	 * 
+	 * public static ActaMartilladoDTO getActaMartilladoDTO(ActaMartillado acta,
+	 * GuiaForestalDTO guiaDTO){
+	 * 
+	 * ActaMartilladoDTO actaDTO = new ActaMartilladoDTO();
+	 * actaDTO.setNroOrden(acta.getNroOrden());
+	 * actaDTO.setProductorForestal(getEntidadDTO(acta.getProductorForestal()));
+	 * actaDTO.setPeriodoForestal(acta.getPeriodoForestal());
+	 * actaDTO.setExpediente(acta.getExpediente());
+	 * actaDTO.setFecha(acta.getFecha());
+	 * actaDTO.setUbicacionZonal(acta.getUbicacionZonal());
+	 * actaDTO.setCantidadUnidades(acta.getCantidadUnidades());
+	 * actaDTO.setTipoTransporte(acta.getTipoTransporte());
+	 * actaDTO.setCantidadMts(acta.getCantidadMts());
+	 * actaDTO.setTamanio(acta.getTamanio()); actaDTO.setGuiaForestal(guiaDTO);
+	 * 
+	 * return actaDTO; }
+	 * 
+	 * public static GuiaForestalDTO getGuiaForestalDTO(GuiaForestalForm form,
+	 * Usuario usuario){
+	 * 
+	 * GuiaForestalDTO guia = new GuiaForestalDTO();
+	 * 
+	 * guia.setNroGuia(form.getNroGuia());
+	 * guia.setFiscalizacion(form.getFiscalizacion());
+	 * guia.setLugarCorte(form.getLugarCorte());
+	 * //guia.setPermisionario(form.getPermisionario());
+	 * guia.setFechaVencimiento
+	 * (Fecha.stringDDMMAAAAToDate(form.getFechaVencimiento()));
+	 * guia.setDistanciaAforoMovil(form.getDistanciaAforoMovil());
+	 * guia.setNroExpediente(form.getNroExpediente());
+	 * guia.setPeriodoForestal(form.getPeriodoForestal());
+	 * guia.setDisposicionNro(form.getDisposicionNro());
+	 * guia.setTipo(form.getTipo()); guia.setEstado(form.getEstado());
+	 * guia.setEspecie(form.getEspecie());
+	 * guia.setVolumenTotal(form.getVolumenTotal());
+	 * guia.setCantidad(form.getCantidad()); guia.setImporte(form.getImporte());
+	 * guia.setAforo(form.getAforo());
+	 * guia.setInspFiscalizacion(form.getInspFiscalizacion());
+	 * guia.setValorAforos(form.getValorAforos());
+	 * guia.setObservaciones(form.getObservaciones());
+	 * guia.setLocalidad(form.getLocalidad());
+	 * guia.setFecha(Fecha.stringDDMMAAAAToDate(form.getFecha()));
+	 * guia.setValesTransporte
+	 * (normalizarValesTransporte(form.getValesTransporte()));
+	 * guia.setBoletasDeposito
+	 * (normalizarBoletasDeposito(form.getBoletasDeposito()));
+	 * guia.setUsuario(usuario);
+	 * 
+	 * return guia; }
+	 * 
+	 * public static GuiaForestalDTO getGuiaForestalDTO(GuiaForestal guia,
+	 * ActaMartilladoDTO actaDTO){
+	 * 
+	 * GuiaForestalDTO guiaDTO = new GuiaForestalDTO();
+	 * 
+	 * guiaDTO.setNroGuia(guia.getNroGuia()); guiaDTO.setFiscalizacion(actaDTO);
+	 * guiaDTO.setLugarCorte(guia.getLugarCorte());
+	 * //guiaDTO.setPermisionario(guia.getPermisionario().getId());
+	 * guiaDTO.setFechaVencimiento(guia.getFechaVencimiento());
+	 * guiaDTO.setDistanciaAforoMovil(guia.getDistanciaAforoMovil());
+	 * guiaDTO.setNroExpediente(guia.getNroExpediente());
+	 * guiaDTO.setPeriodoForestal(guia.getPeriodoForestal());
+	 * guiaDTO.setDisposicionNro(guia.getDisposicionNro());
+	 * guiaDTO.setTipo(guia.getTipo()); guiaDTO.setEstado(guia.getEstado());
+	 * guiaDTO.setEspecie(guia.getEspecie());
+	 * guiaDTO.setVolumenTotal(guia.getVolumenTotal());
+	 * guiaDTO.setCantidad(guia.getCantidad());
+	 * guiaDTO.setImporte(guia.getImporte()); guiaDTO.setAforo(guia.getAforo());
+	 * guiaDTO.setInspFiscalizacion(guia.getInspFiscalizacion());
+	 * guiaDTO.setValorAforos(guia.getValorAforos());
+	 * guiaDTO.setObservaciones(guia.getObservaciones());
+	 * guiaDTO.setLocalidad(guia.getLocalidad());
+	 * guiaDTO.setFecha(guia.getFecha());
+	 * guiaDTO.setValesTransporte(getValesTransporteDTO
+	 * (guia.getValesTransporte(),guiaDTO));
+	 * guiaDTO.setBoletasDeposito(getBoletasDesposito
+	 * (guia.getBoletasDeposito(),guiaDTO));
+	 * guiaDTO.setUsuario(guia.getUsuario());
+	 * 
+	 * return guiaDTO; }
+	 * 
+	 * public static GuiaForestalDTO getGuiaForestalDTO(GuiaForestal guia){
+	 * 
+	 * GuiaForestalDTO guiaDTO = new GuiaForestalDTO();
+	 * 
+	 * guiaDTO.setNroGuia(guia.getNroGuia());
+	 * guiaDTO.setFiscalizacion(getActaMartilladoDTO
+	 * (guia.getActaMartillado(),guiaDTO));
+	 * guiaDTO.setLugarCorte(guia.getLugarCorte());
+	 * //guiaDTO.setPermisionario(guia.getPermisionario().getId());
+	 * guiaDTO.setFechaVencimiento(guia.getFechaVencimiento());
+	 * guiaDTO.setDistanciaAforoMovil(guia.getDistanciaAforoMovil());
+	 * guiaDTO.setNroExpediente(guia.getNroExpediente());
+	 * guiaDTO.setPeriodoForestal(guia.getPeriodoForestal());
+	 * guiaDTO.setDisposicionNro(guia.getDisposicionNro());
+	 * guiaDTO.setTipo(guia.getTipo()); guiaDTO.setEstado(guia.getEstado());
+	 * guiaDTO.setEspecie(guia.getEspecie());
+	 * guiaDTO.setVolumenTotal(guia.getVolumenTotal());
+	 * guiaDTO.setCantidad(guia.getCantidad());
+	 * guiaDTO.setImporte(guia.getImporte()); guiaDTO.setAforo(guia.getAforo());
+	 * guiaDTO.setInspFiscalizacion(guia.getInspFiscalizacion());
+	 * guiaDTO.setValorAforos(guia.getValorAforos());
+	 * guiaDTO.setObservaciones(guia.getObservaciones());
+	 * guiaDTO.setLocalidad(guia.getLocalidad());
+	 * guiaDTO.setFecha(guia.getFecha());
+	 * guiaDTO.setValesTransporte(getValesTransporteDTO
+	 * (guia.getValesTransporte(),guiaDTO));
+	 * guiaDTO.setBoletasDeposito(getBoletasDesposito
+	 * (guia.getBoletasDeposito(),guiaDTO));
+	 * guiaDTO.setUsuario(guia.getUsuario());
+	 * 
+	 * return guiaDTO; }
+	 * 
+	 * public static List<BoletaDepositoDTO>
+	 * getBoletasDesposito(List<BoletaDeposito> boletas, GuiaForestalDTO
+	 * guiaDTO){
+	 * 
+	 * List<BoletaDepositoDTO> boletasDTO = new ArrayList<BoletaDepositoDTO>();
+	 * 
+	 * for (BoletaDeposito boleta : boletas) {
+	 * 
+	 * boletasDTO.add(getBoletaDesposito(boleta,guiaDTO)); }
+	 * 
+	 * return boletasDTO; }
+	 * 
+	 * public static BoletaDepositoDTO getBoletaDesposito(BoletaDeposito boleta,
+	 * GuiaForestalDTO guiaDTO){
+	 * 
+	 * BoletaDepositoDTO boletaDTO = new BoletaDepositoDTO();
+	 * 
+	 * boletaDTO.setArea(boleta.getArea());
+	 * boletaDTO.setConcepto(boleta.getConcepto());
+	 * boletaDTO.setEfectivoCheque(boleta.getEfectivoCheque());
+	 * boletaDTO.setFechaPago
+	 * (Fecha.dateToStringDDMMAAAA(boleta.getFechaPago()));
+	 * boletaDTO.setFechaVencimiento
+	 * (Fecha.dateToStringDDMMAAAA(boleta.getFechaVencimiento()));
+	 * boletaDTO.setMonto(boleta.getMonto());
+	 * boletaDTO.setNumero(boleta.getNumero());
+	 * boletaDTO.setGuiaForestal(guiaDTO);
+	 * 
+	 * return boletaDTO; }
+	 * 
+	 * public static List<ValeTransporteDTO>
+	 * getValesTransporteDTO(List<ValeTransporte> vales, GuiaForestalDTO
+	 * guiaDTO){
+	 * 
+	 * List<ValeTransporteDTO> valesDTO = new ArrayList<ValeTransporteDTO>();
+	 * 
+	 * for (ValeTransporte vale : vales) {
+	 * 
+	 * valesDTO.add(getValeTransporte(vale,guiaDTO)); }
+	 * 
+	 * return valesDTO; }
+	 * 
+	 * public static ValeTransporteDTO getValeTransporte(ValeTransporte vale,
+	 * GuiaForestalDTO guiaDTO){
+	 * 
+	 * ValeTransporteDTO valeDTO = new ValeTransporteDTO();
+	 * 
+	 * valeDTO.setCantidadMts(vale.getCantidadMts());
+	 * valeDTO.setDestino(vale.getDestino());
+	 * valeDTO.setDominio(vale.getDominio());
+	 * valeDTO.setEspecie(vale.getEspecie());
+	 * valeDTO.setFecha(Fecha.dateToStringDDMMAAAA(vale.getFecha()));
+	 * valeDTO.setFechaVencimiento
+	 * (Fecha.dateToStringDDMMAAAA(vale.getFechaVencimiento()));
+	 * valeDTO.setMarca(vale.getMarca());
+	 * valeDTO.setNroPiezas(vale.getNroPiezas());
+	 * valeDTO.setNumero(vale.getNumero()); valeDTO.setOrigen(vale.getOrigen());
+	 * valeDTO.setProducto(vale.getProducto());
+	 * valeDTO.setGuiaForestal(guiaDTO);
+	 * valeDTO.setVehiculo(vale.getVehiculo());
+	 * 
+	 * return valeDTO; }
+	 * 
+	 * private static List<BoletaDepositoDTO>
+	 * normalizarBoletasDeposito(List<BoletaDepositoDTO> boletasDeposito) {
+	 * List<BoletaDepositoDTO> boletasAEliminar = new
+	 * ArrayList<BoletaDepositoDTO>(); for (BoletaDepositoDTO boleta :
+	 * boletasDeposito) {
+	 * 
+	 * if(boleta.getNumero() == 0){ boletasAEliminar.add(boleta); } }
+	 * boletasDeposito.removeAll(boletasAEliminar);
+	 * 
+	 * return boletasDeposito; }
+	 * 
+	 * private static List<ValeTransporteDTO>
+	 * normalizarValesTransporte(List<ValeTransporteDTO> valesTransporte) {
+	 * 
+	 * List<ValeTransporteDTO> valesAEliminar = new
+	 * ArrayList<ValeTransporteDTO>(); for (ValeTransporteDTO vale :
+	 * valesTransporte) {
+	 * 
+	 * if(vale.getNumero() == 0){ valesAEliminar.add(vale); } }
+	 * valesTransporte.removeAll(valesAEliminar);
+	 * 
+	 * return valesTransporte; }
+	 */
 }
Index: src/ar/com/siif/fachada/Ubicacion.java
===================================================================
--- src/ar/com/siif/fachada/Ubicacion.java	(revision 0)
+++ src/ar/com/siif/fachada/Ubicacion.java	(revision 0)
@@ -0,0 +1,90 @@
+package ar.com.siif.fachada;
+
+import ar.com.siif.dto.FiscalizacionDTO;
+import ar.com.siif.negocio.Fiscalizacion;
+import ar.com.siif.negocio.Marcacion;
+import ar.com.siif.negocio.PMF;
+import ar.com.siif.negocio.Rodal;
+import ar.com.siif.negocio.Tranzon;
+
+public class Ubicacion {
+
+	public Ubicacion(IUbicacionFachada pUbicacionFachada) {
+		this.ubicacionFachada = pUbicacionFachada;
+	}
+
+	private IUbicacionFachada ubicacionFachada;
+
+	private Rodal rodal;
+
+	private Marcacion marcacion;
+
+	private Tranzon tranzon;
+
+	private PMF pmf;
+
+	public Rodal getRodal() {
+		return rodal;
+	}
+
+	public void setRodal(Rodal rodal) {
+		this.rodal = rodal;
+	}
+
+	public IUbicacionFachada getUbicacionFachada() {
+		return ubicacionFachada;
+	}
+
+	public void setUbicacionFachada(IUbicacionFachada ubicacionFachada) {
+		this.ubicacionFachada = ubicacionFachada;
+	}
+
+	public Marcacion getMarcacion() {
+		return marcacion;
+	}
+
+	public void setMarcacion(Marcacion marcacion) {
+		this.marcacion = marcacion;
+	}
+
+	public Tranzon getTranzon() {
+		return tranzon;
+	}
+
+	public void setTranzon(Tranzon tranzon) {
+		this.tranzon = tranzon;
+	}
+
+	public PMF getPmf() {
+		return pmf;
+	}
+
+	public void setPmf(PMF pmf) {
+		this.pmf = pmf;
+	}
+
+	public void setUbicacion(FiscalizacionDTO fiscalizacionDTO) {
+		if (fiscalizacionDTO.getRodal().getId() != null) {
+			rodal = ubicacionFachada.getRodal(fiscalizacionDTO.getRodal().getId());
+		}
+		if (fiscalizacionDTO.getMarcacion().getId() != null) {
+			marcacion = ubicacionFachada.getMarcacion(fiscalizacionDTO.getMarcacion().getId());
+		}
+
+		if (fiscalizacionDTO.getTranzon().getId() != null) {
+			tranzon = ubicacionFachada.getTranzon(fiscalizacionDTO.getTranzon().getId());
+		}
+		if (fiscalizacionDTO.getPmf().getId() != null) {
+			pmf = ubicacionFachada.getPMF(fiscalizacionDTO.getPmf().getId());
+		}
+
+	}
+
+	public void setEnFiscalizacion(Fiscalizacion fiscalizacion) {
+		fiscalizacion.setRodal(rodal);
+		fiscalizacion.setTranzon(tranzon);
+		fiscalizacion.setMarcacion(marcacion);
+		fiscalizacion.setPmf(pmf);
+	}
+
+}
Index: src/ar/com/siif/negocio/GuiaForestal.java
===================================================================
--- src/ar/com/siif/negocio/GuiaForestal.java	(revision 290)
+++ src/ar/com/siif/negocio/GuiaForestal.java	(working copy)
@@ -54,6 +54,21 @@
 	private Rodal rodal;
 
 	@ManyToOne()
+	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
+	@JoinColumn(name = "marcacion_fk")
+	private Marcacion maracacion;
+	
+	@ManyToOne()
+	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
+	@JoinColumn(name = "tranzon_fk")
+	private Tranzon tranzon;
+	
+	@ManyToOne()
+	@Cascade(value = { CascadeType.ALL, CascadeType.DELETE_ORPHAN })
+	@JoinColumn(name = "pmf_fk")
+	private PMF pmf;
+	
+	@ManyToOne()
 	@Cascade(value = CascadeType.SAVE_UPDATE)
 	@JoinColumn(name = "entidad_fk")
 	private Entidad productorForestal;
@@ -231,4 +246,27 @@
 		this.anulado = anulado;
 	}
 
+	public Marcacion getMaracacion() {
+		return maracacion;
+	}
+
+	public void setMaracacion(Marcacion maracacion) {
+		this.maracacion = maracacion;
+	}
+
+	public Tranzon getTranzon() {
+		return tranzon;
+	}
+
+	public void setTranzon(Tranzon tranzon) {
+		this.tranzon = tranzon;
+	}
+
+	public PMF getPmf() {
+		return pmf;
+	}
+
+	public void setPmf(PMF pmf) {
+		this.pmf = pmf;
+	}
 }
